libs/datamodel/core/src/transform/ast_to_dml/generator_loader.rs

```diff
@@ -73,10 +74,12 @@ impl GeneratorLoader {
 
         let mut properties: HashMap<String, String> = HashMap::new();
 
-        let binary_targets = match args
-            .get(BINARY_TARGETS_KEY)
-            .map(|v| v.as_array().to_string_from_env_var_vec())
-        {
+        let binary_targets = match args.get(BINARY_TARGETS_KEY).map(|v| {
```

Comment:
Make this a variable, multi-line expression in a match statement is not very nice to read. Name it accordingly.

Reply:
Renamed v to value_validator